app/assets/javascripts/oxalis/view/settings/setting_input_views.js

```diff
@@ -196,7 +196,7 @@ export class NumberInputSetting extends React.PureComponent<NumberInputSettingPr
   }
 }
 
-type VectorInputSettingPropTypes<T: Vector6> = {
```

Comment:
Hmm, there are other specific `...InputSettingsPropTypes` defined in that file, that are all different. It would be a little strange if this one suddenly would get a generic name.
I dropped the `Vector6` as I thought it was too restricting, but it doesn't really matter as it's only used for this one setting. I could add it back and if someone uses this propTypes definition for another setting (e.g. Vector3) he could amend it as `<Vector3 | Vector6>`.